userdocs/src/usage/nodegroup-override-announcement.md

```diff
@@ -37,6 +37,19 @@ command:
       /etc/eks/bootstrap.sh test-override-11 --container-runtime containerd --kubelet-extra-args "--node-labels=${NODE_LABELS}"
 ```
 
+Additionally, you might have to also set `--apiserver-endpoint` and `--b64-cluster-ca`. For private nodegroups set these
+as follows:
```

Comment:
For nodegroups that have no outbound internet access, you'll need to supply `--apiserver-endpoint` and `--b64-cluster-ca` to the bootstrap script. These values can be obtained using `aws eks describe-cluster --name <cluster>`.